config-overrides.js

```diff
@@ -1,10 +1,19 @@
-const { override, fixBabelImports, addWebpackPlugin } = require('customize-cra');
+const { override, fixBabelImports, addWebpackPlugin,addLessLoader } = require('customize-cra');
 const AntdDayjsWebpackPlugin = require('antd-dayjs-webpack-plugin');
 module.exports = override(
+   addWebpackPlugin(new AntdDayjsWebpackPlugin()),
+  addLessLoader({
+    lessOptions:{
+    javascriptEnabled: true,
+    modifyVars:{
+      '@primary-color':'#b175ca',
+    }
```

Comment:
You may prettify this file.

Also, you can import less variables in js. This eliminates the need of using `modifyVars`. Please refer to https://frontend-cookbook.ack.ee/pages/implementation/SharingVariables.html for more information.

Reply:
BTW, you may have to move all variables to a separate `custom.less` file. Otherwise antd may be imported multiple times (not sure about this).

Reply:
there are some colors not applying my customization if this is not included.

Reply:
and u can't use less variables in js.

Reply:
Alright. You may also try using `require('????/theme.js')["primary-color"]` here to reduce redundancy.